profile/README.md

```diff
@@ -0,0 +1,41 @@
+# OpenTimelineIO
+[![OpenTimelineIO](images/OpenTimelineIO@3xDark.png)](http://opentimeline.io)
+
+The OpenTimelineIO github organization is a federation of repos relating to the
+[OpenTimelineIO core](https://github.com/AcademySoftwareFoundation/OpenTimelineIO)
+project.
+
+Repositories stored in this organization include adapters, examples, and more.
```

Comment:
Should we mention language bindings? Or are they covered by more?

Reply:
Good call!